website/docs/components/rich-tooltip/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,205 @@
+
+The most common options are: `top`, `bottom`, `left`, `right`.
+
+![Placement examples of the Rich Tooltip](/assets/components/rich-tooltip/rich-tooltip-placement.png)
+
+The Rich Tooltip supports additional placement options: `top-start`, `top-end`, `bottom-start`, `bottom-end`, `left-start`, `left-end`, `right-start`,and `right-end`.
+
+![Additional placements of the Rich Tooltip](/assets/components/rich-tooltip/rich-tooltip-additional-placements.png)
+
+!!! Info
+
+The placement property is _relative_ to the element that toggles the component, rather than the direction of the pointer. E.g., `placement=left` will render the component to the _left_ of the toggle, but the pointer will be pointing to the right.
+!!!
+
+### Collision detection
+
+The Rich Tooltip supports collision detection, meaning the `placement` property might be overridden depending on the position of the component relative to the edge of the viewport. This ensures that the component doesn't extend outside of the viewport while still being associated with the toggle element.
+
+![Example of collision detection in the Rich Tooltip](/assets/components/rich-tooltip/rich-tooltip-collision-detection.png)
+
+## Offset
+
+The default recommended distance between the toggle and the Rich Tooltip pointer is 4px.
+
+![Default spacing for the Rich Tooltip](/assets/components/rich-tooltip/rich-tooltip-offset-spacing.png)
+
+!!! Dont
+
+Don’t add extra offset if the Rich Tooltip would block important information, appear disconnected from the element it is meant to provide information for, or cause confusion for the user.
+
+![Example of adjusting the offset of the Rich Tooltip](/assets/components/rich-tooltip/rich-tooltip-offset-dont.png)
+!!!
+
+## Interaction
+
+The Rich Tooltip supports two interaction methods: a `On hover / On focus` ("soft") interaction and a `On click` interaction. Which one to use depends on:
+
+- The type and complexity of content displayed by the component.
+- Whether _temporarily persisting_ the content in the UI is necessary or beneficial to the user.
+
+Independent of which interaction is used, the Rich Tooltip can be dismissed by clicking outside of the component or with the `ESC` key.
+
+## Toggle
+
+### Default
+
+By default, we provide a [Toggle](/components/rich-tooltip?tab=specifications#toggle-1) component which should be used in the majority of cases along with the Rich Tooltip. This component ensures that the toggle is perceivable, visually consistent, and can be used inline with other content or standalone as part of the layout flow.
+
+#### How to use
+
+Consider these guidelines when using the default Toggle component:
+
+!!! Do
+
+When used in a block of text, use the "text only"/"no icon" variant, as the icon can unnecessarily break the reading flow of the content.
+
+![Text only underline](/assets/components/rich-tooltip/rich-tooltip-toggle-text-only.png)
```

Comment:
@jorytindall in the image, the color of the toggle text is not correct, it should be `foreground-strong` independently from the surrounding text. Right?

Reply:
Upon inspection, I think many other (all?) images are using `Foreground/Primary` color for the text/icon instead of `Foreground/Strong`

Reply:
Hmmmm, that must have been something that didn't cross over into the website assets. I'll make sure all of these are updated.

Reply:
Ok I believe I've updated all of these.